### DISCUSSION TITLE
Sort by date listed

### DISCUSSION THREAD

**Discussion by landerr9290** [rgriebl/brickstore#538] [Ideas]
Is there a way to sort by date listed after importing store inventory?  Would be helpful for turnover.

**Comment by ZZJHONS** [comment]
There is the column Added but seems BS only shows data if you listed using BL interface, if you listed the lot by API it seems to be blank. But checking the XML download seems the field is there. May be a bug.

But you can use the column Lot ID as those are created in ascending order whenever you list anything. But to know the exact date you would need to check the website.
